geniza/corpus/templates/corpus/snippets/document_header.html

```diff
@@ -16,5 +16,5 @@
     {# Translators: Default label when document does not have a type #}
     {% translate 'Unknown type' as unknown_type %}
     <span class="doctype">{{ document.doctype|default:unknown_type }}</span>
-    <span class="shelfmark">{{ document.shelfmark_display }}</span>
+    <span class="shelfmark">{{ document.shelfmark }}</span>
```

Comment:
I think we need a formatted version so you can wrap individual shelfmarks with a tag and set white space not to wrap in the styles.

not sure we need a new method, though — maybe just iterate and generate here in the template? but should probably do something similar for search results; maybe a template snippet would work for both cases

Reply:
Done in f3aacc7 👍